### DISCUSSION TITLE
Fail to build from git

### DISCUSSION THREAD

**Discussion by camvinh** [emgucv/emgucv#514] [Q&A]
Hello everyone! I want to compile the EmguCV v4.4.0, and I downloaded the OpenCV v4.4.0 and placed it in the EmguCV\opencv , and the OpenCV Contrib v4.4.0 is placed in EmguCV\opencv_contrib. Then run EmguCV\platforms\windows\Build_Binary_x86-64_doc.bat, but the build fails. I don't know which step I did wrong.

**Comment by emgucv** [comment]
If you are building on windows, you can follow the instructions here:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Building_from_Git

The instruction for checking out Emgu CV from github can be found here:
https://www.emgu.com/wiki/index.php/GIT

You do not need to download open cv / opencv contrib v4.4.0. The folders are initialized by the command:
`git submodule update --init --recursive`
It will checkout a modified version (not the official one) of opencv and opencv contrib for Emgu CV.
